src/components/scrollTop.js

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useState } from "react";
+
+// https://www.coderomeos.org/scroll-to-top-of-the-page-a-simple-react-component
+// the source code
+export default function ScrollToTop() {
+  const [isVisible, setIsVisible] = useState(false);
+
+  // Show button when page is scorlled upto given distance
+  const toggleVisibility = () => {
+    if (window.pageYOffset > 300) {
```

Comment:
Would this need to be changed for different screen or window sizes?

Reply:
Yeah, I think so, I will make a change.